tests/loggers/test_all.py

```diff
@@ -68,10 +68,10 @@ def log_metrics(self, metrics, step):
 
 @pytest.mark.parametrize("logger_class", [
     TensorBoardLogger,
-    CometLogger,
     MLFlowLogger,
     NeptuneLogger,
     TestTubeLogger,
+    # CometLogger,  # TODO: add this one
```

Comment:
what happened that this is not valid anymore?

Reply:
see #1682 

i see this has been closed, does that mean we no longer need to pickle loggers? if so, should we remove the test?

Reply:
strange, I think it shall be pickable...

Reply:
feel free to reproduce with the code sample i included in #1682 :)